app/controllers/api/v1/search_terms_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V1
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/c919a05a12101ae07b4c21bf82ccf33697056e9e/app/controllers/api/v1/search_terms_controller.rb#L2"></span>Has the name 'v1'</td>
    </tr>
  </tbody>
</table>